### PR TITLE
Fix testing pod's xcode 7.3 compatiblity

### PR DESCRIPTION
**Pull request by cprime**

### CODE REVIEW

**Comment by Benuuu** [comment]
:shipit:

**Comment by loganwright** [comment]
:shipit:

File: SwiftWisdomTests/Testing/XCTest/XCTest+Optionals.swift [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Foundation
 import XCTest
 
-public func ip_XCTAssertEqualOptional<T: Any where T: Equatable>(@autoclosure expression1: () -> T?, @autoclosure expression2: () -> T?, _ message: String? = nil, file: String = __FILE__, line: UInt = __LINE__) {
+public func ip_XCTAssertEqualOptional<T: Any where T: Equatable>(@autoclosure expression1: () -> T?, @autoclosure expression2: () -> T?, _ message: String? = nil, file: StaticString = #file, line: UInt = #line) {
```
Comment (loganwright) [inline SwiftWisdomTests/Testing/XCTest/XCTest+Optionals.swift:4]: Let's remove `ip_` on public functions since they're namespaced to module

**Comment by paulrolfe** [comment]
@cprime Is this PR still waiting on anything?

**Comment by paulrolfe** [comment]
Re: Logan's comment. He makes a good point, maybe we shouldn't be using the `ip_` prefix anywhere anymore. Should be considered for changes that happen for shift to swift3.

**Pull request closed without merging**